data_model/src/isi.rs

```diff
@@ -143,43 +143,42 @@ pub mod model {
         /// `Upgrade` variant.
         Upgrade(UpgradeBox),
     }
+}
+
+mod shared {
+    // NOTE: instructions in this module don't have to be made opaque with `model!`
+    // because they are never shared between client and server(http)/host(wasm)
+
+    use super::*;
 
     /// Generic instruction to set key value at the object.
-    #[derive(Debug, Clone, Constructor, Decode, Encode, Deserialize, Serialize, Getters)]
-    #[getset(get = "pub")]
-    pub struct SetKeyValue<O, K, V>
+    #[derive(Debug, Clone)]
```

Comment:
I don't understand how did you remove encoding and serialization derives if they should be used in boxed structure...

Reply:
Also I think it's better to keep `Constructor`

Reply:
> I don't understand how did you remove encoding and serialization derives if they should be used in boxed structure...

These structs are never serialized

>  Also I think it's better to keep Constructor

`Constructor` is there to make structs opaque. These are not opaque so I removed it